specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -1643,6 +1643,45 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/invoices/{invoiceName}": {
```

Comment:
We now have /billingAccounts/x/invoices/{invoiceName} *and* /billingAccounts/x/billingProfiles/y/invoices/{invoiceName}, and a caller will need to figure out which one to call. Really awkward, but I understand how we got here for our 2019 preview API. In our 2020 GA APIs this is all converged and makes much more sense.